README.md

```diff
@@ -1,76 +1,76 @@
-| BREV      | Missing              | Missing               | &check;      | &check;       |
-| BRFLD     | Missing              | Missing               | &check;      | &check;       |
-| BRSA      | Missing              | Missing               | &check;      | &check;       |
-| BRSPD     | Missing              | Missing               | &check;      | &check;       |
-| BSHDAT    | &cross;              | &cross;               | &cross;      | &cross;       |
-| BSMAT     | Missing              | Missing               | &check;      | &check;       |
-| BTUNDA    | Missing              | Missing               | &check;      | &check;       |
-| BUCT      | &check;              | &check;               | &check;      | &check;       |
-| INI       | Missing              | Missing               | &check;      | &check;       |
-| LC        | &check;              | &check;               | &check;      | &check;       |
-| PKG       | &check;              | &check;               | &check;      | &check;       |
-| TOC       | &check;              | &check;               | &check;      | &check;       |
-| TXT       | &check;              | &check;               | &check;      | &check;       |
-| WEBM      | Missing              | Missing               | &cross;      | &cross;       |
+| Format    | Samus Returns (Read) | Samus Returns (Write) | Dread (Read) | Dread (Write) | Samus Returns (Purpose) | Samus Returns (Notes) |
+|-----------|----------------------|-----------------------|--------------|---------------|-------------------------|-----------------------|
+| BAPD      | Missing              | Missing               | &check;      | &check;       |                         |						|
+| BCCAM     | &cross;              | &cross;               | &cross;      | &cross;       | Camera Objects	        |						|
+| BCLGT     | &cross;              | &cross;               | Missing      | Missing       | Lighting (?)	        |						|
+| BCMDL     | &cross;              | &cross;               | &check;      | &cross;       | 3D Models	            |						|
+| BCPTL     | &cross;              | &cross;               | &cross;      | &cross;       | PTL (?)	                | 						|
+| BCSKLA    | &check;              | &check;               | &check;      | &check;       | Skeleton Animation	    |						|
+| BCTEX     | &cross;              | &cross;               | &check;      | &cross;       | Texture File	        | Texture files can be dumped, customized, and re-written using popular 3DS emulation software.|
+| BCURV     | Missing              | Missing               | &cross;      | &cross;       |                         |						|
+| BCUT      | &cross;              | &cross;               | Missing      | Missing       |	UT (?)                  |						|
+| BCWAV     | &cross;              | &cross;               | Missing      | Missing       | WAV Audio File			|						|
```

Comment:
I would not call it "WAV Audio File", as that way it can be mistaken for a [WAV audio file](https://en.wikipedia.org/wiki/WAV)

Reply:
Would it just be called a Sound File in that case?

Reply:
sound file or audio file, whatever floats your boat more.